tests/unit/slider/slider_methods.js

```diff
@@ -100,8 +100,90 @@ test( "value", function() {
+
+	element.slider( "values", 0, 50 );
+	equal( element.slider( "values", 0 ), 50, "Values (index,value) - set first handle" );
+
+	element.slider( "values", 1, 75 );
+	equal( element.slider( "values", 1 ), 75, "Values (index,value) - set second handle" );
+
+	element.slider( "values", [ 50, 75 ] );
+	element.slider( "values", 0, 5 );
+	equal( element.slider( "values", 0 ), 10, "Values (index,value) - 1st handle restricted properly, against min" );
+
+	element.slider( "values", 1, 5 );
+	equal( element.slider( "values", 0 ), 10, "Values (index,value) - 2nd handle restricted properly, against min" );
+
+	element.slider( "values", [ 50, 75 ] );
+	element.slider( "values", 0, 110 );
+	equal( element.slider( "values", 0 ), 75, "Values (index,value) - 1st handle restricted properly, against max" );
+
+	element.slider( "values", 1, 110 );
+	equal( element.slider( "values", 1 ), 100, "Values (index,value) - 2nd handle restricted properly, against max" );
+
+	element.slider( "values", [ 50, 75 ] );
+	element.slider( "values", 0, 90 );
+	equal( element.slider( "values", 0 ), 75, "Values (index,value) - 1st handle restricted properly, against 2nd handle" );
+
+	element.slider( "values", [ 50, 75 ] );
+	element.slider( "values", 1, 45 );
+	equal( element.slider( "values", 1 ), 50, "Values (index,value) - 2nd handle restricted properly, against 1st handle" );
+	
+	element.slider( "values", [ 50, 75 ] );
+	element.slider( "values", [ 50, 10 ] );
+	deepEqual( element.slider( "values" ), [ 50, 50 ], "Values (array) - Setting both handles - 2nd Handle resets to 1st" );
+
+	element.slider( "values", [ 50, 75 ] );
+	element.slider( "values", [ 75, 50 ] );
+	deepEqual( element.slider( "values" ), [ 75, 75 ], "Values (array) - Setting both handles - 2nd Handle resets to 1st" );
+
+	element.slider( "destroy" );
+	element = $( "<div></div>" ).slider({
+		range: true,
+		min: 1,
+		max: 2,
+		step: 0.2
+	});
+
+	element.slider( "values", [ 1.4, 1.2] );
+	deepEqual( element.slider( "values" ), [ 1.4 , 1.4 ], "Values (array) - Floating Point -  2nd value restricted" );
+
+	element.slider( "destroy" );
+	element = $( "<div></div>" ).slider({
+		range: true,
+		min: 1.1,
+		max: 2,
+		step: 0.2
+	});
+
+	element.slider( "values", [ 0.1, 2.2] );
+	deepEqual( element.slider( "values" ), [ 1.1, 1.9 ], "Values (array) - Floating Point - uneven step - restricted min max" );
+
+	element.slider( "values", [1.5, 1.3] );
+	deepEqual( element.slider( "values" ), [ 1.5, 1.5 ], "Values (array) - Floating Point - uneven step - 2nd Handle restricted" );
+
+	element.slider( "values", [1.5, 1.5] );
+	element.slider( "values", 1, 1.3 );
+	deepEqual( element.slider( "values" ), [ 1.5, 1.5 ], "Values (index,value) - Floating Point - uneven step - 2nd Handle restricted" );
+
+	element.slider( "values", [1.5, 1.5] );
+	element.slider( "values", 0, 1.6 );
+	deepEqual( element.slider( "values" ), [ 1.5, 1.5 ], "Values (index,value) - Floating Point - uneven step - 1st Handle restricted" );
```

Comment:
Why are we doing a whole slew of floating point tests for boundary restrictions?